apps/explorer/lib/explorer/chain.ex

```diff
@@ -7874,13 +7926,46 @@ defmodule Explorer.Chain do
     end)
     |> Multi.run(:delete_celo_pending, fn _, _ ->
       success
-      |> Enum.each(fn reward -> Chain.delete_celo_pending_epoch_operation(reward.block_hash) end)
+      |> Enum.each(fn reward ->
+        Chain.falsify_or_delete_celo_pending_epoch_operation(reward.block_hash, :fetch_epoch_rewards)
+      end)
+
+      {:ok, success}
+    end)
+    |> Explorer.Repo.transaction()
+  end
+
+  def import_group_votes_and_delete_pending_celo_epoch_operations(import_params, success) do
```

Comment:
Let's stop using Ecto.Multi in the Chain context and use the import runner functionality instead. Database operations are not related to Blockchain functionality and adding features here is adding to the mess that is this file.